prusti-server/src/server.rs

```diff
@@ -54,35 +54,36 @@ where
     let viper = Arc::new(Viper::new_with_args(config::extra_jvm_args()));
     stopwatch.finish();
 
-    let build_verification_request_handler = |viper_arc: Arc<Viper>| {
+    let cache_loc = config::cache_dir() + "data.json";
```

Comment:
`let cache_loc = PathBuf::from(config::cache_dir()).join("data.json");` is more robust, e.g. if `cache_dir` doesn't end with a `/`.